h/templates/includes/share_widget.html.jinja2

```diff
@@ -0,0 +1,55 @@
+{% macro share_widget() -%}
+
+<div class="share-widget">
+  <div class="share-widget-target">
+    <span class="share-widget-target__label">Share:</span>
+    <a share-target-href="https://twitter.com/intent/tweet?url={href}&hashtags=annotated"
+      target="_blank"
```

Comment:
Hmm, we open each of these share links in a new window but I notice that the Facebook, Twitter and G+ UIs that appear all look like they are designed to appear in a popup, and the example buttons at the bottom of https://dev.twitter.com/web/tweet-button do